toolchain/lc-compile-ffi-java/src/LAYOUT.b

```diff
@@ -0,0 +1,2 @@
+\  { AdvanceCurrentPosition(1); }
+\t {  }
```

Comment:
You need to `AdvanceCurrentPosition` for tab characters, or the incorrect column number will be reported in error and warning messages.